ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml

```diff
@@ -413,5 +413,95 @@ ViewData["Title"] = "Notification - Clinical Details";
             </div>
         </div>
     </date-comparison>
+    <br/>
+
+    <h2>Treatment Options</h2>
+    <validate-multiple model="ClinicalDetails" :properties="['IsShortCourseTreatment', 'IsMDRTreatment']" inline-template>
+        <div>
+             @{ var ShortCourseGroupState = !Model.IsValid("ClinicalDetails.IsShortCourseTreatment") ? Error : Standard; }
+            <nhs-form-group nhs-form-group-type="@ShortCourseGroupState" v-bind:class="{ 'nhsuk-form-group--error': hasError}">
+                <nhs-fieldset>
+                    <nhs-fieldset-legend nhs-legend-size="Standard">Has Started Short Course Treatment?</nhs-fieldset-legend>
+                    <span ref="errorField[0]" nhs-span-type="ErrorMessage" asp-validation-for="ClinicalDetails.IsShortCourseTreatment"></span>
+                    <nhs-radios nhs-radios-type="Standard">
+                        <nhs-radios-item>
+                            <input nhs-input-type="Radios" asp-for="ClinicalDetails.IsShortCourseTreatment" id="short-course-yes" type="radio" value="true" ref="input[0]" v-on:change="validate"/>
```

Comment:
Most other fields we've used `v-on:blur`, is there a reason to break this consistency? Should we move all fields to `change`?

Reply:
On-change is a bit nicer with radio buttons as you get instant feedback when you click on the option.. happy to go with v-blur if you think it would be better to not show the error until you move away from the field

Reply:
Nope, that seems like a good enough reason!